server/test/unit/server/managers/repo/test_unit_association_query.py

```diff
@@ -673,385 +636,3 @@ def _assert_unit_integrity(self, unit):
         self.assertTrue(unit['metadata']['md_1'] is not None)
         self.assertTrue(unit['metadata']['md_2'] is not None)
         self.assertTrue(unit['metadata']['md_3'] is not None)
-
-
-class GetUnitsStressTest(base.PulpServerTests):
```

Comment:
I didn't realize this was here before, good catch.

The git log indicates this was updated in 2011-ish, aside from refactors and PEP8 updates.